packages/core/src/shadow/PipelinePass.ts

```diff
@@ -0,0 +1,21 @@
+import { Engine } from "../Engine";
```

Comment:
this file should migrate to `core/src/RenderPipeline`

Reply:
done